lib/std/os.zig

```diff
@@ -44,6 +44,32 @@ pub const windows = @import("os/windows.zig");
 pub const posix_spawn = @import("os/posix_spawn.zig");
 pub const ptrace = @import("os/ptrace.zig");
 
+/// Pipe read side
```

Comment:
Move into child_process. A namespace `Pipe` sounds nice, but I don't know things to move inside except for init and the constants.